index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="pages/styles/style.css" rel="stylesheet">
+
+</head>
+<body>
+  <h1> Roza Ajema's Portfolio</h1>
+  <nav class = "nav">
+<ul>
+  <li><a href="index.html">Home</a> </li>
+  <li><a href="pages/about.html">About</a> </li>
+  <li><a href="pages/portfolio.html">Portfolio</a></li>
+</ul>
+  </nav>
+<div class="wrapper">
+<section class = "me">
+<div>
+  <h2> Currently Roza has three last names: [Roza Ajema, Roza Cedar, Roza C16]</h2>
+  <p class="paragraph">I was born and raised in Ethiopia, and moved to Seattle 4 years ago. I have a beautiful 4 years old daughter. 
+  I spend most of my time with my daughter, socially engaging with family, friends and church community.
+  I enjoy visiting ancient places and unique cultures. Professionally, I am heading to become a full stack developer. 
+  I enjoy coding, I feel happy when I fix bugs eventhough its very tedious. 
+  My life principles are easy: <em><b>Happiness is a choice and forgiving is a key.</b></em></p></div>
+  <div>
+    <figure>
+  <img id="roza" src="pages/images/Roza-pic.jpg" alt="Roza with Haset">
+  <figcaption>Selfie by Roza</figcaption>
+  </figure>
+</div>
+</section>
```

Comment:
The indentation here makes it hard to read but I think you have an extra `<div>` you don't need for styling.